lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1549,6 +1555,34 @@ def download_pe_conf_if_master(host)
             end
           end
         end
+
+        # Method to install just the agent nodes
+        # This method can be called only after installing PE on all other nodes
+        # including infrastructure nodes, loadbalancer, hubs and spokes
+        # @param [Array] agent only nodes from Beaker hosts
+        # @param [Hash] opts The Beaker options hash
+        def install_agents_only_on(agent_nodes, opts)
```

Comment:
@shaigy Is there a reason you didn't directly copy paste? I would prefer to keep the step blocks around.

Reply:
Ah, ok. Will update it shortly

Reply:
Thanks, thats the last change I have, then this and the beaker-pe-lei one is good to go if you've re-ran the tests and verified it all works still.

Reply:
Done! Validated with couple of configurations. I will add a comment with test result.